app/src/main/java/org/simple/clinic/AppDatabase.kt

```diff
@@ -217,6 +217,7 @@ abstract class AppDatabase : RoomDatabase() {
       val facilityIdsInCurrentSyncGroup = facilityDao().facilityIdsInSyncGroup(currentFacility.syncGroup)
 
       patientDao().deletePatientsNotInFacilities(facilityIdsInCurrentSyncGroup)
+      bloodPressureDao().deleteWithoutLinkedPatient()
```

Comment:
In what scenario would BP measurements be linked without patient? Shouldn't this be deleted in the use-case where the patient is removed?

Reply:
Yes.